src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,28 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String constructDiamond(int diamondDiameter) {
+		if (diamondDiameter <= 0 || diamondDiameter % 2 == 0) {
 			return null;
 		}
-		StringBuilder builder = new StringBuilder(new String(make(n, n)));
-		for (int i = n - 2; i > 0; i -= 2) {
-			char[] chars = make(n, i);
-			builder.insert(0, chars);
-			builder.append(chars);
+		StringBuilder diamondBuilder = new StringBuilder(new String(createDiamondRow(diamondDiameter, diamondDiameter)));
+		for (int space = diamondDiameter - 2; space > 0; space -= 2) {
```

Comment:
Die Bezeichnung space kann verwirrend sein, da sie in diesem Kontext "Leerzeichen" bedeuten kann und nicht Platz den diese Reihe einnimmt.